docs/source/user-guide/dataframe.md

```diff
@@ -95,6 +95,7 @@ These methods execute the logical plan represented by the DataFrame and either c
 | write_csv                  | Execute this DataFrame and write the results to disk in CSV format.                                                         |
 | write_json                 | Execute this DataFrame and write the results to disk in JSON format.                                                        |
 | write_parquet              | Execute this DataFrame and write the results to disk in Parquet format.                                                     |
+| write_table                | Execute this DataFrame and write the results via the insert_into method of the registered TableProvider                     |
```

Comment:
lgtm @devinjdangelo 

But I also just realized we dont have `write_parquet` covered with tests. Creating a followup

Reply:
Followup PR open for this as well #8531